src/loaders/zarrLoader.js

```diff
@@ -49,6 +49,41 @@ export default class ZarrLoader {
     };
   }
 
+  setSelection(selections) {
```

Comment:
What is this function for? I don't see where `this._selections` is used.

Reply:
This is somewhat vestigal from my inital attempts to nail down setting selections on loaders. I ended up using this as a starting point for `vitessce-image-loader`. I could remove here since it is unused.

Reply:
removed in favor of what is implemented in `vitessce-image-loader`